lib/gemfury/command/app.rb

```diff
@@ -292,4 +292,19 @@ def die!(msg, err = nil, command = nil)
     shell.say %Q(#{err.class.name}: #{err}\n#{err.backtrace.join("\n")}) if err && ENV['DEBUG']
     exit(1)
   end
+
+  def time_ago(tm)
```

Comment:
I was thinking that we always show the timestamp, and then append a user-friendly hint if it's less than 24 hours ago.  Something like this, what do you think:

```
 def time_ago(tm)
    out = tm.strftime('%F %R')
    in_secs = Time.now - tm

    out << if in_secs < 60
      ' (%d secs ago)' % in_secs
    elsif in_secs < 3600
      ' (%s mins ago)' % (in_secs / 60).ceil
    elsif in_secs < (3600 * 24)
      ' (%s hours ago)' % (in_secs / 3600).ceil
    end

    return out
end
```